### DISCUSSION TITLE
请问最近有更新库的计划吗？

### DISCUSSION THREAD

**Discussion by skyfox6688** [oldwinter/knowledge-garden#85] [建议]
我看最新的是2022，不知道你的工作流改变没

**Comment by oldwinter** [comment]
哈哈，说实话，真的忙起来的时候，什么复杂工作流都要失效，没时间。我现在可能就dailynote 工作流还一直持续进行，其他工作流都不怎么用了。

关于本库，因为最近使用canvas比较多，计划等Obsidian官方的支持canvas 的publish出来以后，更新一个版本。
